externs/mediaelements.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+/**
+ * @fileoverview Externs for HTML Media Elements.
+ *
+ * @externs
+ */
+
+
+/**
+ * @override
+ */
+HTMLMediaElement.prototype.play = function() {};
```

Comment:
This file is not needed.  This is already known by the compiler (the error is about duplicate definitions).